output/typescript/types.ts

```diff
@@ -901,6 +901,7 @@ export interface ReindexRethrottleResponse {
 }
 
 export interface RenderSearchTemplateRequest extends RequestBase {
+  id?: Id
```

Comment:
Shouldn't this be in nested in a `path_parts`? It's confusing to see a `body` property and free-floating properties (same comment for other similar additions).